pkgs/desktops/budgie/budgie-analogue-clock-applet/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, vala
+, budgie
+, gtk3
+, libpeas
+}:
+
+stdenv.mkDerivation rec {
+  name = "analogue-clock-applet";
```

`name` defaults to `${pname}-${version}` (the src needs upd accordingly)

```suggestion
  pname = "analogue-clock-applet";
```